qemu/tests/win_virtio_driver_update_by_installer.py

```diff
@@ -96,6 +108,9 @@ def change_virtio_media(cdrom_virtio):
             static_ip_address = utils_net.get_guest_ip_addr(session_serial, virtio_nic_mac, os_type='windows')
             if static_ip_address != params["static_ip"]:
                 test.fail("Failed to setup static ip,current ip is %s" % static_ip_address)
```

Comment:
@heywji These lines also could be included into the function.

Reply:
OK

Reply:
Done.

Test Result: pass
```
Host_RHEL.m9.u4.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.Win10.x86_64.io-github-autotest-qemu.win_virtio_driver_install_by_installer.driver_update.from_old_installer.q35: PASS (960.66 s)
```